app/boards/arm/nice_nano/nice_nano_v2.dts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include "nice_nano.dtsi"
+
+/ {
+	ext-power {
+		compatible = "zmk,ext-power-generic";
+		label = "EXT_POWER";
+		control-gpios = <&gpio0 13 GPIO_ACTIVE_HIGH>;
+		init-delay-ms = <10>;
+	};
+
+	vbatt {
+		compatible = "zmk,battery-voltage-divider";
+		label = "BATTERY";
+		io-channels = <&adc (0x0D - 1)>;
```

Comment:
Is this effectively the VDDH hack that's similar to the driver from @joelspadin in #750 ?

Reply:
This is correct. It's not too much of a hack, as the VDDH internal ADC works similarly to the AIN pins just with fewer options.

Reply:
Yeah, the only really hacky things about it are the seemingly arbitrary numbers needed to get it to work. 0x0D is the number of the channel we want, but the driver adds one to it so we have to subtract one. Then we don't really have a voltage divider but the ADC needs to be rescaled, so we just use the voltage divider resistances to get the multiplier we need.